harvard-huit/Harvard-Geospatial-Library
# Remove Extra Filters From Left Navigation Bar

**Remove Extra Filters From Left Navigation Bar.**
* * *

# What does this Pull Request do?
An upgrade to Blacklight added the selected search filters back on to the left navigation, above the "Limit Your Search" heading. This PR removes those duplicate filters. 

# How should this be tested?

A description of what steps someone could take to:
* Build your container off of `main` and perform a search. You should see your search filters appear above the "Limit Your Search" heading, along with a link that says "Start Over."
* Pull in the `fix-filter` branch and rebuild the container.
* Perform another search and the duplicate filter and "Start Over." link no longer appear above "Limit Your Search".

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@enriquediaz @dl-maura @mferrarini
